README.md

```diff
@@ -50,6 +54,29 @@ sls deploy
 ```
 5. 👀 Head to the CloudWatch section of the AWS Console to check out your new dashboards 📊 and alarms ⏰ !
 
+
+## Getting Started - CloudFormation Macro
```

Comment:
Getting Started - *AWS SAM/CloudFormation*

This method uses the SLIC Watch CloudFormation macro. It is very simple to add this macro as a transform to your SAM or CloudFormation template, but first you must ensure that the SLIC Watch macro has been deployed to the same AWS account as your application.